draft-ietf-shmoo-remote-fee.md

```diff
@@ -116,7 +116,8 @@ and other roles in the IETF.
 These and other running costs of the IETF are also cross-financed by income generated through
 meeting fees. The intention of this document and the principle stated herein is not to make participation
 free for everyone but to always have a free option that can be used without any barriers other than
-the registration procedure itself.
+the registration procedure itself. As long as there are still enough paying participants to cover
+the base costs, additional participant can effectively be added with adding expenses.
```

Comment:
Is it strictly necessary to add this text "As long as there are still enough paying participants to cover
the base costs, additional participant can effectively be added with adding expenses."? It seems possible in the future that some of our operating costs, including software license fees, may be variable costs that vary by the number of attendees or number of simultaneous users.

Reply:
It's of course not strictly necessary but this point was made in the discussion at the last meeting and I think it's a good point to add. Actually the more important point is that there should not be a maximum limit of fee waivers and given the costs structure we have right now there seems no good reason to do so as long as we have enough paid registration. It not really the case that we need explicitly sponsorship for each waiver. However, you are right that the costs can change in future, so maybe rephrase that somehow.

Reply:
Is the intention of this to also address Brian's comment about not bankrupting the LLC?

Reply:
Not necessarily. I thought that is implicitly covered in the misuse section below but we could make it more explicit by adding another sentence (and I can add it to this PR):

OLD
Aggregated data on the number and percentage of free registrations used should be published,
as this will permit analysis of the use and change in use over time of the free registration option without
 revealing personal information.

NEW
The LLC is responsible to  ensure the financial stability of the IETF and therefore should monitor trends in the use of the free participating option that could endanger the viability of the IETF. Aggregated data on the number and percentage of free registrations used should be published,
as this will permit analysis of the use and change in use over time of the free registration option without
 revealing personal information.

Reply:
actually I created a separate PR: #20